### PR TITLE
Prestissimo docker setup WIP

### PR DESCRIPTION
**Pull request by minhancao**
Prestissimo docker setup WIP

### CODE REVIEW

File: prestissimo/workers/1/node.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node.environment=2509758e5a6160ddf8c3014651b64006eec5213a
+node.data-dir=/var/lib/presto/data
```
Comment (czentgr) [inline prestissimo/workers/1/node.properties]: On Linux/Mac writing to this dir requires root permissions. Perhaps it should be something like
`/tmp/presto/data`.
On Mac root is also needed. So docker-compose seems to have root privileges. But for a play area probably better to put this in tmp.

File: prestissimo/coordinator/node.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+node.environment=2509758e5a6160ddf8c3014651b64006eec5213a
+node.data-dir=/var/lib/presto/data
```
Comment (czentgr) [inline prestissimo/coordinator/node.properties]: Perhaps it should be something like
`/tmp/presto/data`

File: prestissimo/workers/1/node.properties (hunk 1 above)
Comment (minhancao) [reply prestissimo/workers/1/node.properties]: Done

File: prestissimo/coordinator/node.properties (hunk 2 above)
Comment (minhancao) [reply prestissimo/coordinator/node.properties]: Done

**Review by czentgr — APPROVED** [review]

**Pull request merged**